### DISCUSSION TITLE
Export list item

### DISCUSSION THREAD

**Discussion by vegas75** [lairdubois/lairdubois-opencutlist-sketchup-extension#586] [Q&A]
Hello! Does this parameter have a variable for the export list?
![Screenshot](https://github.com/lairdubois/lairdubois-opencutlist-sketchup-extension/assets/73779261/23a2f54d-42a2-4552-a874-fe894be4e73f)

**Comment by bbeaulant** [comment]
Hi @vegas75,

No, currently there's no variable to export this attribute.
In what case do you need to export it ?

**Reply by vegas75** [reply]
Thanks.
I am a technologist at a furniture enterprise. This column in the export table could save the time when I edit the application of cutting the material ))
